### PR TITLE
Use correct join game method

### LINKED ISSUE #1259: Discord Integration has some bugs
When a game starts it still says "wating" and joining doe not work properly sating "wrong password".

### PR DESCRIPTION
**Pull request by 1-alex98**
The other bug that PLAYING is not displayed or games not removed is fixed in a different PR, that reworks code parsing game changes from the server. See PR #1261...
Fixes #1259

### CODE REVIEW

**Review by Geosearchef — DISMISSED** [review]

**Comment by 1-alex98** [comment]
@micheljung  pls review

File: src/main/java/com/faforever/client/game/JoinGameHelper.java [hunk 1]
```diff
@@ -28,7 +32,8 @@
 import static java.util.Arrays.asList;
 
 @Component
-@Scope
+@Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/game/JoinGameHelper.java]: Singleton is the default scope.

File: src/main/java/com/faforever/client/discord/DiscordRichPresenceService.java [hunk 2]
```diff
@@ -93,6 +93,8 @@ public void updatePlayedGameTo(Game game, int currentPlayerId, String currentPla
 
       if (game.getStartTime() != null) {
         discordRichPresence.setStartTimestamps(game.getStartTime().toEpochMilli());
+      } else if (game.getStatus() == GameStatus.PLAYING) {
+        discordRichPresence.setStartTimestamps(Instant.now().toEpochMilli());
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/discord/DiscordRichPresenceService.java:97]: This is confusing me.
a) When would this case happen?
b) As you can not join a running game, the start timestamp is still the start time of the game, isn't it?

**Review by Brutus5000 — CHANGES_REQUESTED** [review]

File: src/main/java/com/faforever/client/game/JoinGameHelper.java (hunk 1 above)
Comment (1-alex98) [reply src/main/java/com/faforever/client/game/JoinGameHelper.java]: ok

File: src/main/java/com/faforever/client/discord/DiscordRichPresenceService.java (hunk 2 above)
Comment (1-alex98) [reply src/main/java/com/faforever/client/discord/DiscordRichPresenceService.java:97]: Well the start time is sometimes set by the server after the change in game status therefore it is not updated....

Comment (Brutus5000) [reply src/main/java/com/faforever/client/discord/DiscordRichPresenceService.java:97]: Stupid server issues

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**